sdk/communication/AzureCommunicationChat/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.1.0 (Unreleased)
+Updated AzureCommunicationChat Version
```

Comment:
Is this planned to be a GA release? If so, you will likely need an archboard review.

Also, you should probably call out the new API version.

Reply:
Yes, it is planned to be a GA release for this.  The release date is not decided yet.  I will need to get the API view approved. What do you mean by `call out the new API version`? 
FYI, @juancamilor

Reply:
"Updated AzureCommunicationChat Version" is not a useful changelog message. It could be something like "Updated service API version to 2021-09-07."